### PR TITLE
Updated bulk update guidance to use just LA 

### PR DESCRIPTION
**Pull request by ScottDawsonDfE**
- renaming LA (Name), and removing LA Code. Also applied lint/reformat of the file

### CODE REVIEW

**Comment by ScottDawsonDfE** [comment]
Note: most of the cahnges are fixing the formatting of the file. Unfortunately it makes spotting the two bits of real code change difficult
Line 189 in the new file
And lines 251-254 removed from the old file

**Comment by RogerHowellDfE** [comment]
> Note: most of the changes are fixing the formatting of the file. Unfortunately it makes spotting the two bits of real code change difficult Line 189 in the new file And lines 251-254 removed from the old file

To make this easier to review, could you split this into two separate commits - one to do the change, and a second commit to do the reformat? (or the other way around)

File: Web/Edubase.Web.UI/Views/Guidance/EstablishmentBulkUpdate.cshtml [hunk 1]
```diff
@@ -32,840 +32,837 @@
...
+    <td class="govuk-table__cell" data-label="Equivalent field name in app">County (name)</td>
+</tr>
+<tr class="govuk-table__row">
+    <td class="govuk-table__cell" data-label="Database name (use this for your upload file)">Postcode</td>
+    <td class="govuk-table__cell" data-label="Equivalent field name in app">Postcode</td>
+</tr>
+<tr class="govuk-table__row">
+    <td class="govuk-table__cell" data-label="Database name (use this for your upload file)">LA</td>
+    <td class="govuk-table__cell" data-label="Equivalent field name in app">Local authority</td>
+</tr>
```
Comment (RogerHowellDfE) [inline Web/Edubase.Web.UI/Views/Guidance/EstablishmentBulkUpdate.cshtml:191]: It is now ambiguous which value should be provided (name or ID or code).

Reading through the bug comments I _think_ it's the LA code which is expected.

Either way, elaborating here would be helpful I think.

**Review by RogerHowellDfE — CHANGES_REQUESTED** [review]

**Review by BarryGibney — APPROVED** [review]

File: Web/Edubase.Web.UI/Views/Guidance/EstablishmentBulkUpdate.cshtml (hunk 1 above)
Comment (ScottDawsonDfE) [reply Web/Edubase.Web.UI/Views/Guidance/EstablishmentBulkUpdate.cshtml:191]: Discussed with @RogerHowellDfE 
This is ambiguous as the value updates the LA using either Code or Name.
There are much bigger questions about the guidance as updating LA also updates other fields

**Review by RogerHowellDfE — APPROVED** [review]

**Pull request merged**